src/main/kotlin/app/extractors/Languages.kt

```diff
@@ -0,0 +1,108 @@
+// Copyright 2018 Sourcerer Inc. All Rights Reserved.
+// Author: Alexander Surkov (alex@sourcerer.io)
+
+package app.extractors
+
+object LG {
```

Comment:
![image](https://user-images.githubusercontent.com/29913247/40841122-072a61ac-65b2-11e8-8ba7-e2ae385c042b.png)

Reply:
"Langs" would be more readable and short enough. It's also a good practice to name a file the same as a class inside.

Reply:
According to https://www.allacronyms.com/language/abbreviated language may be abbreviated to 'lg'. However if you prefer 'Lang' abbreviation, then 'Lang' look slightly better with me than its plural form.